app/utils/emoji_utils.test.js

```diff
@@ -218,3 +218,34 @@ describe('hasEmojisOnly with empty and mixed emojis', () => {
         });
     }
 });
+
+describe('doesMatchNamedEmoji', () => {
+    const testCases = [{
+        input: ':named_emoji:',
+        output: true,
+    }, {
+        input: 'named_emoji',
+        output: false,
+    }, {
+        input: ':named_emoji',
+        output: false,
+    }, {
+        input: 'named_emoji:',
+        output: false,
+    }, {
+        input: '::named_emoji:',
+        output: false,
+    }, {
+        input: 'named emoji',
+        output: false,
+    }, {
+        input: ':named emoji:',
+        output: false,
```

Comment:
Non-blocking but what about some cases with punctuation or extra letters on the end? e.g. `:named_emoji:!`, `:named_emoji:aa`

Reply:
Those should return false as well.  I added those test cases.  Thanks!